dali/base/dautils.cpp

```diff
@@ -116,7 +116,7 @@ class CMultiDLFN
             CDfsLogicalFileName lfn;
             lfn.setAllowWild(true);
             lfn.set(s);
-            dlfns.push_back(lfn);
+            dlfns.push_back(std::move(lfn));
```

Comment:
Agreed. I can add a constructor with emplace_back to reduce code size.
I did not evaluate the actual performance, other than to identify that this was less than optimal.